socorro/external/es/connection_context.py

```diff
@@ -4,7 +4,7 @@
 
 import contextlib
 
-import elasticsearch
+import elasticsearch_1_9_0 as elasticsearch
```

Comment:
How are we thinking about switching between two different versions of the libraries?

When I was working on this, I was thinking I'd build a shim module and then change all the imports and things to use the shim module and it would abstract away the differences between the library versions. So then this line would be something like:

```python
from socorro.external.es.es_shim import elasticsearch
```

And the shim would be something like this for this PR step:

```python
import elasticsearch_1_9_0 as elasticsearch
```

and later would evolve to something more complex like:

```python
if os.environ.get("ES_VERSION") == "legacy":
    import elasticsearch_1_9_0 as elasticsearch
else:
    import elasticsearch
```

Are you thinking of doing that? If not, what are you thinking of doing? I ask because it affects how we change all the import lines.

Reply:
I talked with @relud in Zoom. We're not planning to do a shim. Instead, we'll have both live at the same time, but do a configuration-based feature-flipping kind of thing for the webapp and crontabber. For now, the plan is to have all the imports at the top of all the webapp and crontabber Python files, but maybe we'll switch that to a shim later.

Given that, we can leave this and other imports as they are in this PR.